tests/test_blockshelper.py

```diff
@@ -19,10 +19,10 @@ def test_get_first_block_after(self):
         self.sci.get_block_number.return_value = 8
         self.sci.get_block_by_number.side_effect = lambda number: blocks[number]
 
-        assert self.bh.get_first_block_after(0) == blocks[0]
-        assert self.bh.get_first_block_after(11) == blocks[1]
-        assert self.bh.get_first_block_after(21) == blocks[1]
-        assert self.bh.get_first_block_after(55) == blocks[5]
-        assert self.bh.get_first_block_after(98) == blocks[8]
-        with self.assertRaisesRegex(ValueError, 'no blocks after'):
-            self.bh.get_first_block_after(99)
+        assert self.bh.get_latest_existing_block_at(0) == blocks[0]
```

Comment:
Please rename test as well.

Reply:
Changed, added fixup and I have already sqashed it into one